packages/geo/src/lib/map/geolocate-button/geolocate-button.component.ts

```diff
@@ -13,6 +13,7 @@ import { IgoMap } from '../shared/map';
 })
 export class GeolocateButtonComponent implements AfterContentInit, OnDestroy {
   private tracking$$: Subscription;
+  private isTemporaryDisableFollowPositionToSwitch: Boolean;
```

Comment:
Boolean vs boolean, on devrait toujours utiliser le dernier qui est le [type primitif](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean)

Reply:
C'est corrigé